dotmap/__init__.py

```diff
@@ -39,6 +39,7 @@ def __init__(self, *args, **kwargs):
             for k,v in src:
                 if self._prevent_method_masking and k in reserved_keys:
                     raise KeyError('"{}" is reserved'.format(k))
+                k = self.__fix_ilegal_name__(k)
```

Comment:
Can you rename this to `__fix_illegal_name__`? Specifically, `ilegal` is spelled incorrectly.

Reply:
sorry for the typo, fixed.